pkg/ccpatch/register.go

```diff
@@ -0,0 +1,7 @@
+package ccpatch
+
+import (
+	// Import substitution packages, to make them available to the user
+	_ "github.com/ionos-cloud/crossplane-provider-ionoscloud/pkg/ccpatch/substitution/ipv4address"
+	_ "github.com/ionos-cloud/crossplane-provider-ionoscloud/pkg/ccpatch/substitution/ipv6address"
+)
```

Comment:
I don't understand why this is needed?

Reply:
this is a modular architecture and allows you to register new modules on the fly, by prefixing with `_` the Init() inside of the package is called, and the handler is registered.

This pattern is kinda common in go

Reply:
Ok, I got it now. Thank you